src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,15 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+
+        double a = (2 * Math.sin(x - Math.PI / 6) * calculateB(z)) / (1.0 / 2 + Math.pow(Math.sin(y), 2.0));
+        return a;
     }
 
     private static double calculateB(double z) {
-        return 0;
+
+        double sqr_z = z * z;
```

Comment:
please avoid `C`-style variable names. In java, there is [Java Code convention](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf) that you should follow.